ui/apps/platform/src/Containers/Policies/Table/PoliciesTable.tsx

```diff
@@ -81,6 +88,16 @@ const columns = [
     },
 ];
 
+const defaultPolicyLabel = 'System';
+const userPolicyLabel = 'User';
```

Comment:
Although not a blocker to merge, a helper function like `getOriginLabel(isDefault: boolean)` instead of constants would avoid repetition 2 times below and then 1 more in rendering of ternary expression.